cfme/v2v/migration_settings.py

```diff
@@ -25,28 +27,74 @@
 from widgetastic_manageiq import WaitTab
 
 
-class MigrationSettingsView(BaseLoggedInPage):
+class MigrationThrottling(Updateable, NavigatableMixin):
+
+    def __init__(self, appliance, migration_settings):
+        self.appliance = appliance
+        self.migration_settings = migration_settings
+
+    def set_max_migration_per_conv_host(self, migration_per_conv_host):
+        """Migration throttling for conversion host
+        Args:
+            migration_per_conv_host: Integer
+        """
+        view = navigate_to(self.migration_settings, 'MigrationSettings')
+        view.tabs.migration_throttling.fill({'migration_per_conv_host': migration_per_conv_host})
+        view.tabs.migration_throttling.apply_btn.click()
```

Comment:
Would it be a good idea to also provide ability to read currently set value(a property or `get()`) ? So that we can assert the value we have set is correct.

Reply:
yes , We can read the value set in UI , Will cover that in separate test .

Reply:
That would be good candidate for UI tests, but would it be possible to add that method along with this class while we are at it? We know we might need it/might wanna test it in future right?

Reply:
@kedark3 : Although I was not targeting to cover it in this test/PR , I have added the method .There will be more tests coming for this UI .